package.json

```diff
@@ -27,15 +27,25 @@
     "grunt-svgmin": "^3.1.2",
     "grunt-usemin": "^3.1.1",
     "grunt-wiredep": "^2.0.0",
+    "jasmine": "^2.4.1",
+    "jasmine-core": "^2.4.1",
     "jshint-stylish": "^2.1.0",
+    "karma": "^1.1.1",
+    "karma-jasmine": "^1.0.2",
+    "karma-junit-reporter": "^1.1.0",
+    "karma-phantomjs-launcher": "^1.0.1",
+    "karma-spec-reporter": "0.0.26",
     "load-grunt-tasks": "^3.4.0",
     "time-grunt": "^1.3.0"
   },
   "engines": {
     "node": ">=0.10.0"
   },
   "scripts": {
-    "test": "grunt test"
+    "postinstall": "bower install",
+    "pretest": "npm install",
+    "test": "karma start test/karma.conf.js",
+    "test-single-run": "karma start test/karma.conf.js --single-run"
```

Comment:
@msabramo Maybe this is a dump question but how can I trigger this `test-single-run`? Just `npm test-single-run` won't work. As you can [see here](https://travis-ci.org/kwk/docker-registry-frontend/builds/145763756), travis will stuck after a while when running `npm test`. I know that this is normal and that it allows me open various browsers to run my tests.